components/button/floating-buttons.js

```diff
@@ -0,0 +1,236 @@
+import 'lit-media-query/lit-media-query.js';
+import { css, html, LitElement } from 'lit-element/lit-element.js';
+import { ButtonMixin } from './button-mixin.js';
+import { buttonStyles } from './button-styles.js';
+import { labelStyles } from '../typography/styles.js';
+
+class FloatingButtons extends ButtonMixin(LitElement) {
```

Comment:
These don't need to extend `ButtonMixin` since they're not actually buttons -- they wrap the buttons.

Reply:
@cjmorrison-d2l : assuming we keep the button spacing styles (via `::slotted`), I think we will need the `RtlMixin`.